src/api/tesseractmain.cpp

```diff
@@ -69,6 +69,31 @@ static void Win32WarningHandler(const char* module, const char* fmt,
 }
 
 #endif /* HAVE_TIFFIO_H */
+
+class AutoWin32ConsoleOutputCP {
+ public:
+  explicit AutoWin32ConsoleOutputCP(UINT codeCP) : changed_(false) {
+    auto oldCP = GetConsoleOutputCP();
```

Comment:
Maybe it's possible to remove this? Just always set and unset, even if the same CP.